demo/server.ts

```diff
@@ -1,5 +1,5 @@
 import { APP_BASE_HREF } from '@angular/common';
-import { CommonEngine } from '@angular/ssr';
+import { CommonEngine } from '@angular/ssr/node';
```

Comment:
18 -> 19 breaking change - we don't use content of this file for Netlify deplyoments (if it's using `express`), so this change is not needed to deploy - but still it make sense to update the "default" one

Reply:
This is more or less content of `server.ts` files users get when scaffolding new projects for node@19 ( https://github.com/netlify/angular-runtime/blob/feat/support-v19/tests/fixtures/angular-19-common-engine/src/server.ts ) is fully actual one. So this is what user projects would look like more-or-less. This file is not actually used for Netlify deployments tho